python/revng/api/exceptions.py

```diff
@@ -5,3 +5,7 @@
 
 class RevngException(Exception):
     pass
+
+
+class RevngCException(Exception):
+    pass
```

Comment:
What is this?

Reply:
It's a separate exception that is thrown when an `error_container` contains an error, this is to distinguish it from `RevngException` that's thrown if an error occurs on the python side

Reply:
`RevngC` has a quite specific meaning in rev.ng. Can you find a better name?
Maybe `RevngPipelineCException`?

Reply:
Renamed to `RevngPipelineCException`